src/asl.ts

```diff
@@ -3345,6 +3362,51 @@ export class ASL {
     );
   }
 
+  /**
+   * In some cases, variable can be batch declared as state parameters.
+   * This method is an alternative to {@link evalDecl} which can evaluate multiple {@link ParameterDecl}s
+   * with initial values into a parameter object and any state to be run after (which setup any bound variables.)
+   *
+   * @returns a tuple tuple
```

Comment:
> a tuple tuple

not helpful

Reply:
The useful part is next. did not mean `tuple tuple` though. Updated.

![image](https://user-images.githubusercontent.com/289213/183427394-911399b0-3a6b-4c95-93a5-b979775cdec0.png)